pkg/policyviolation/builder.go

```diff
@@ -3,55 +3,46 @@ package policyviolation
 import (
 	"fmt"
 
+	"github.com/golang/glog"
 	kyverno "github.com/nirmata/kyverno/pkg/api/kyverno/v1"
-	client "github.com/nirmata/kyverno/pkg/dclient"
+	"github.com/nirmata/kyverno/pkg/engine/response"
 )
 
+func GeneratePVsFromEngineResponse(ers []response.EngineResponse) (pvInfos []Info) {
```

Comment:
As discussed before, the `policyviolation` package is now dependent on `engine` (engine/response).
I believe the conclusion was to decouple the packages.